pkg/patterns/addon/pkg/status/kstatus.go

```diff
@@ -44,21 +44,25 @@ func (k *kstatusAggregator) BuildStatus(ctx context.Context, info *declarative.S
 		}
 	}
 
+	// Here we augment each deployment manifests abnormal conditions and determine the declarativeObject's present
+	// condition from these conditions.
+	// https://github.com/kubernetes-sigs/cli-utils/tree/master/pkg/kstatus#conditions
+	var abnormalConditions []status.Condition
+
 	if shouldComputeHealthFromObjects {
 		statusMap := make(map[status.Status]bool)
 		for _, object := range info.Manifest.Items {
 			gvk := object.GroupVersionKind()
 			nn := object.NamespacedName()
 
-			log := log.WithValues("kind", gvk.Kind).WithValues("name", nn.Name).WithValues("namespace", nn.Namespace)
+			log = log.WithValues("kind", gvk.Kind).WithValues("name", nn.Name).WithValues("namespace", nn.Namespace)
```

Comment:
I think we need to keep the shadowing (i.e. `log := log.WithValues...`) ; we're creating a new log here, rather than assigning back to the original log value.  Once we exit the loop, log will no longer include `kind` / `name` / `namespace`. 

This code might be too subtle for its own good...  Sorry if I got this wrong before!

Reply:
Fixed. Thank you so much for pointing it out.